keras/engine/topology.py

```diff
@@ -2923,6 +2923,29 @@ def preprocess_weights_for_loading(layer, weights,
                                                     (2, 3, 1, 0))
                 weights = [kernel, recurrent_kernel, bias]
 
+        if layer.__class__.__name__ in ['Model', 'Sequential']:
+            new_weights = []
+            # trainable weights
+            for sublayer in layer.layers:
+                num_weights = len(sublayer.trainable_weights)
+                if num_weights  > 0:
+                    new_weights.extend(preprocess_weights_for_loading(layer=sublayer,
```

Comment:
Start the keyword arguments on the next line, indented with 4 spaces, to avoid overly long lines